src/layout/constraint.rs

```diff
@@ -323,40 +482,213 @@ mod tests {
         assert_eq!(Constraint::from_mins(vec![1, 2, 3]), expected);
     }
 
+    #[test]
+    fn from_fills() {
+        let expected = [
+            Constraint::Fill(1),
+            Constraint::Fill(2),
+            Constraint::Fill(3),
+        ];
+        assert_eq!(Constraint::from_fills([1, 2, 3]), expected);
+        assert_eq!(Constraint::from_fills(vec![1, 2, 3]), expected);
+    }
+
     #[test]
     fn apply() {
-        assert_eq!(Constraint::Percentage(0).apply(100), 0);
-        assert_eq!(Constraint::Percentage(50).apply(100), 50);
-        assert_eq!(Constraint::Percentage(100).apply(100), 100);
-        assert_eq!(Constraint::Percentage(200).apply(100), 100);
-        assert_eq!(Constraint::Percentage(u16::MAX).apply(100), 100);
+        assert_eq!(
+            {
+                #[allow(deprecated)]
```

Comment:
Slap this on the test itself.

Reply:
Thanks! I tried it on the `assert_eq!` and that didn't work.

Reply:
Does it work on the test function?